scene/resources/mesh_data_tool.h

```diff
@@ -37,6 +37,12 @@ class MeshDataTool : public RefCounted {
 	GDCLASS(MeshDataTool, RefCounted);
 
 	int format = 0;
+	struct CustomVertexData {
+		Color fd;
+		uint8_t ud[8];
+		//uint32_t type;
+	};
+
```

Comment:
The fact that you made this a struct and not a union also points towards something...